api/service.go

```diff
@@ -601,6 +694,19 @@ func (s *Service) getCollection(creds thread.Credentials, collectionName string)
 	return collection, nil
 }
 
-func getCredentials(c *pb.Credentials) (thread.Credentials, error) {
-	return thread.NewSignedCredsFromBytes(c.ThreadID, c.PubKey, c.Signature)
+func getAuth(header *pb.Header, body proto.Message) (*thread.Auth, error) {
```

Comment:
Yea, curious how this would look as request metadata

Reply:
Nice, good call... I totally forgot about metadata amidst all the other changes.

Reply:
👍